src/VisualLinter/Helpers/StylelintHelper.cs

```diff
@@ -0,0 +1,292 @@
+﻿using Microsoft.VisualStudio.Shell;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace jwldnr.VisualLinter.Helpers
+{
+    internal static class StylelintHelper
```

Comment:
This whole class is basically a copy paste of the `EslintHelper.cs` class. I'd like there to be shared `LinterHelper.cs` class with common methods used by both `EslintHelper.cs` and `StylelintHelper.cs`.

The `internal` modifier methods can remain, but most private ones should be moved to `LinterHelper.cs`. See below comments for examples: